starter-code/index.js

```diff
@@ -1,17 +1,55 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = []
+    this.length = 0
```

Comment:
la length se esperaba que pusieras la length de los items, es decir this.length=this.items.length